### PR TITLE
Adding info for render function

### PR DESCRIPTION
**Pull request by carstenmichel**
Adding further information about the render function.

### CODE REVIEW

File: docs/lab/assets/frontend/React_Tables.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ Another coffee script example demonstrates the usage of a custom render function
 ```
 This function checks for `null` first and if `field` is not null it adds ` %` to the `field` value.
 
+Reminder: The render function is not ment to be used for calculations. Since the render function is allowed to return all kinds of elements, including further HTML or React elements it will not be evaluated for sorting. Its only purpose it to have a more user friendly graphical representation of the value.
```
Comment (sbatin) [inline docs/lab/assets/frontend/React_Tables.md]: looks like a typo here "is not ment" -> "is not meant"

**Review by sbatin — CHANGES_REQUESTED** [review]

File: docs/lab/assets/frontend/React_Tables.md (hunk 1 above)
Comment (carstenmichel) [reply docs/lab/assets/frontend/React_Tables.md]: corrected, thanks

**Comment by carstenmichel** [comment]
corrected the typo

**Review by sbatin — APPROVED** [review]
LGTM

**Comment by carstenmichel** [comment]
Hello @sbatin thanks for approving. Will you merge as well?

**Comment by sbatin** [comment]
@carstenmichel merged

**Pull request merged**